tests/Wallabag/CoreBundle/Helper/ContentProxyTest.php

```diff
@@ -933,20 +1000,26 @@ public function dataForChangedUrl()
                 'http://example.com/hello-world',
                 null,
                 'example.com',
+                true,
             ],
         ];
     }
 
     /**
      * @dataProvider dataForChangedUrl
      */
-    public function testWithChangedUrl($entry_url, $origin_url, $content_url, $expected_entry_url, $expected_origin_url, $expected_domain)
+    public function testWithChangedUrl($entry_url, $origin_url, $content_url, $expected_entry_url, $expected_origin_url, $expected_domain, $processor_result)
     {
         $tagger = $this->getTaggerMock();
         $tagger->expects($this->once())
             ->method('tag');
 
-        $proxy = new ContentProxy((new Graby()), $tagger, $this->getValidator(), $this->getLogger(), $this->fetchingErrorMessage, true);
+        $ruleBasedIgnoreOriginProcessor = $this->getRuleBasedIgnoreOriginProcessorMock();
+        $ruleBasedIgnoreOriginProcessor->expects($this->once())
+            ->method('process')
+            ->willReturn($processor_result);
+
+        $proxy = new ContentProxy((new Graby()), $tagger, $ruleBasedIgnoreOriginProcessor, $this->getValidator(), $this->getLogger(), $this->fetchingErrorMessage, true);
```

Comment:
I'm not really happy with this test because the processor is mocked.
But as its constructor would require a non-mocked RulerZ instance I don't see how to instantiate it.